libr/anal/bb.c

```diff
@@ -171,8 +172,10 @@ R_API void r_anal_bb_set_offset(RAnalBlock *bb, int i, ut16 v) {
 	// the offset of the instruction 0 is not stored because always 0
 	if (i > 0) {
 		if (i >= bb->n_op_pos) {
+			int offset = bb->n_op_pos;
 			bb->n_op_pos = i * 2;
 			bb->op_pos = realloc (bb->op_pos, bb->n_op_pos * sizeof (*bb->op_pos));
+			memset (bb->op_pos + offset, 0, (bb->n_op_pos - offset) * sizeof (*bb->op_pos));
```

Comment:
no need to memset just one part. the whole buffer should be cleaned up, or filled when `read`